src/commonMain/kotlin/exchange.dydx.abacus/calculator/TriggerOrdersInputCalculator.kt

```diff
@@ -284,7 +290,7 @@ internal class TriggerOrdersInputCalculator(val parser: ParserProtocol) {
             OrderType.TakeProfitMarket, OrderType.StopMarket -> {
                 val triggerPrice =
                     parser.asDouble(parser.value(triggerOrder, "price.triggerPrice"))
-                val majorMarket = MAJOR_MARKETS.contains(parser.asString(triggerOrder["marketId"]))
```

Comment:
"marketId" is located at the one level above instead of here. @moo-onthelawn

Reply:
ah `marketId` doesn't exist in `data class TriggerOrder`?

Reply:
Yeah, so the calculation is not correct :-(